diff --git a/node_modules/docusaurus-plugin-typedoc-api/lib/components/ApiIndex.js b/node_modules/docusaurus-plugin-typedoc-api/lib/components/ApiIndex.js
index 36a5f2c..269e164 100644
--- a/node_modules/docusaurus-plugin-typedoc-api/lib/components/ApiIndex.js
+++ b/node_modules/docusaurus-plugin-typedoc-api/lib/components/ApiIndex.js
@@ -23,7 +23,7 @@ const React__default = /*#__PURE__*/_interopDefault(React);
 const Link__default = /*#__PURE__*/_interopDefault(Link);
 
 function addVersionToUrl(url, latestVersion, preferredVersion) {
-  if (!url.match(/api\/(\d\.\d\.\d)/) && !url.includes('api/next') && preferredVersion && preferredVersion.name !== latestVersion.name) {
+  if (!url.match(/api\/(\d+\.\d+)/) && !url.includes('api/next') && preferredVersion && preferredVersion.name !== latestVersion.name) {
     const version = preferredVersion.name === 'current' ? 'next' : preferredVersion.name;
 
     if (url.endsWith('/api')) {
